components/script/dom/htmllinkelement.rs

```diff
@@ -217,6 +218,15 @@ impl VirtualMethods for HTMLLinkElement {
 
 
 impl HTMLLinkElement {
+    pub fn increment_pending_loads_count(&self) {
+        self.pending_loads.set(self.pending_loads.get() + 1)
+    }
+
+    pub fn decrement_pending_loads_count(&self) -> bool {
+        self.pending_loads.set(self.pending_loads.get() - 1);
```

Comment:
I think this is unused now, I switched to use `load_finished`, so I'll just get rid of it.